packages/cli/src/fingerprint/globber.ts

```diff
@@ -84,8 +112,6 @@ class Globber extends EventEmitter {
 interface Globber {
   on(event: 'end', listener: () => void): this;
   on(event: 'add' | 'change' | 'unlink', listener: (path: string) => void): this;
-  emit(event: 'end'): boolean;
-  emit(event: 'add' | 'change' | 'unlink', path: string): boolean;
```

Comment:
Why remove this part of the type? It's useful for the consumers.

Reply:
`emit` is called by Globber itself, not consumers/subscribers. What's useful about it code outside this class?

Reply:
You're right. I think it's still useful for typechecking the calls inside the Globber.

Reply:
Fair enough. I'll put it back.